resources/agent_python.rb

```diff
@@ -34,3 +34,4 @@
 attribute :browser_monitoring_auto_instrument, :kind_of => [TrueClass, FalseClass], :default => true
 attribute :cross_application_tracer_enable, :kind_of => [TrueClass, FalseClass], :default => true
 attribute :feature_flag, :kind_of => String, :default => nil
+attribute :python, :kind_of => String, :default => nil
```

Comment:
Yes, here's the cookbook's [.kitchen.yml](https://github.com/djoos-cookbooks/newrelic/blob/master/.kitchen.yml)...